EventFilter/CTPPSRawToDigi/python/ctppsDiamondRawToDigi_cfi.py

```diff
@@ -1,11 +1,20 @@
 import FWCore.ParameterSet.Config as cms
 
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
+from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
+
 from EventFilter.CTPPSRawToDigi.totemVFATRawToDigi_cfi import totemVFATRawToDigi
 
 ctppsDiamondRawToDigi = totemVFATRawToDigi.clone(
     subSystem = cms.string('TimingDiamond'),
+    fedIds = cms.vuint32(579, 581, 582, 583),
```

Comment:
might be useful to specify with a comment where these numbers (also in the other files) are coming from.

Reply:
Not much to say from where these numbers are coming: these are Fed numbers assigned in the HW, as declared in
DataFormats/FEDRawData/interface/FEDNumbering.h